app/static/styles.css

```diff
@@ -0,0 +1,209 @@
+:root {
```

Comment:
I suggest using some UI CDN, instead of writing raw CSS on your own. Example: Bootstrap

You can be more creative in using another package. Don't think I am suggesting Bootstrap.